recipes-sdk/aws-sdk-cpp/aws-sdk-cpp_1.9.185.bb

```diff
@@ -0,0 +1,39 @@
+# -*- mode: Conf; -*-
+SUMMARY = "AWS C++ SDK"
+HOMEPAGE = "https://github.com/aws/aws-sdk-cpp"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=e3fc50a88d0a364313df4b21ef20c29e"
+
+SRC_URI = " \
+    gitsm://github.com/aws/aws-sdk-cpp.git;protocol=https;branch=main;tag=1.9.185 \
```

Comment:
Though most amazon SDK's use tags, we are switching our yocto recipes to strictly use `SRC_REV`. Can we switch to that here? I looks like the rev is `9c7d9735e4fd9f018bfab79ef0d5890bbf2b58f3` for this release.

Reply:
Thanks!